pages/[bip].js

```diff
@@ -48,7 +48,7 @@ export async function getStaticProps({ params }) {
     let data = markdown.toJSON(file)
     if (!data.body) throw new Error(`Failed to parse markdown page for ${bipFile}.md`)
     const title = `BIP${(params.bip)} - ${file.substring(file.indexOf('Title: ') + 7, file.indexOf('Author: '))}`
-    const originalURL = `https://github.com/bitcoin/bips/blob/master/bip-${params.bip.padStart(4, '0')}.mediawiki`
+    const originalURL = `https://github.com/bitcoin/bips/blob/master/bip-${bipFile}.mediawiki`

```

Comment:
You have to remove "bip-" in the string. `bipFile` already provide the full file name `bip-0118`
At the moment it returns this `https://github.com/bitcoin/bips/blob/master/bip-bip-0118.mediawiki`
It can be merge after this